app/Http/Models/User.php

```diff
@@ -20,8 +26,14 @@ class User extends Model
         'password',
         'remember_token',
     ];
+
     public $timestamps = true;
 
+    public function isRole()
```

Comment:
```suggestion
    public function roleName()
```
Không cần thiết phải viêt thành 1 function ntn, dùng ở đâu thì gọi ở đó

Reply:
ah ha, chị thấy em để User hợp lí ko chị, dùng cái auth cho user luôn ạ